### DISCUSSION TITLE
Support for other RDBMSs

### DISCUSSION THREAD

**Discussion by andymarden** [Chia-Network/chia-blockchain#16794] [Ideas]
The SQLite database file is getting really big now. I am suffering farming slowdowns which I think is to do  with this - missed signage points etc. Disk I/O shows no issues - I think it is purely the size of the file and the access patterns.

At this size, chia should really move to something like mysql or postgresql. I can't imagine the SQL is complicated or non-standard.

Suggest creating at least the ability for advanced users to specify their own database connection so they can migrate to something like this.

**Comment by BrandtH22** [comment]
Hey @andymarden , as a heads up we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia).

As for the database option, we at CNI have determined that sqlite's flat files are the best option at current for the database. Other community members have created prs for migrating to postgre or mysql and anyone is free to develop their own version of the chia client that utilizes these solutions (https://github.com/Chia-Network/chia-blockchain/pull/10176).

As for the database being the cause of missed signage points, we have not yet seen this issue arise.

Can you confirm:
- Is the database on a fast ssd?
- Were there any network or hardware issues around the missing signage points?
- Do you have any errors or warnings in the logs?
